streams/src/test/java/org/apache/kafka/streams/integration/RepartitionWithMergeOptimizingIntegrationTest.java

```diff
@@ -50,29 +55,33 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
 
 @Category({IntegrationTest.class})
```

Comment:
nit: same as above remove annotation and change test name since it's no longer an integration test.